### DISCUSSION TITLE
Weird Text Rendering

### DISCUSSION THREAD

**Discussion by zhao-li** [ohmyzsh/ohmyzsh#11011] [Help]
Hi Folks,

I recently updated my brew stuff which may have included tmux, zsh, and oh-my-zsh.

And now I have this weird text rendering problem, that I don't know how to describe.  I thought the attached ascii GIF would describe it better than I can.  I'm not even sure what to Google for this type of behavior.

My dot files are here: https://github.com/zhao-li/dots

This effect only shows up when I have zsh/oh-my-zsh with tmux.  Without tmux, everything seems fine.

I upgraded 2 machines and both have this behavior.  The 3rd machine I have not updated yet and is behaving as expected.

Any ideas for where I should look and troubleshoot?

Thank you for your time 🙏
Alan

![zsh_tmux](https://user-images.githubusercontent.com/2177879/175190850-121b6349-e56a-4cfb-b0ae-270662e787d0.gif)

**Comment by carlosala** [comment]
Which is the output of `echo $LANG` for you?

**Reply by zhao-li** [reply]
Thank you for your help 🙏
The output is `en_US.UTF-8`
Please let me know what else I can provide.  Thanks again 😄

**Reply by carlosala** [reply]
I was expecting not to be UTF-8. I have never seen this before, could you surf a bit on our issues to see if there's something similar?

**Reply by zhao-li** [reply]
Bummer.  But thanks for the tip.
I did try looking through the issues but didn't see anything.
Is there a term that you can think of to describe this behavior other than "weird text rendering"?
Thanks again for your help 🙏

**Reply by carlosala** [reply]
Maybe extra characters displayed?

**Reply by zhao-li** [reply]
Thanks for the tips 💯

**Comment by rwmitchell** [comment]
For the next person, how did you fix it?

**Reply by zhao-li** [reply]
I am still looking for a solution. I think it is only happening on Tmux 3.3a but not Tmux 3.2a. Maybe that is helpful for others. 🤞
